src/app/hero-search/hero-search.component.ts

```diff
@@ -0,0 +1,33 @@
+import { HeroService } from './../hero.service';
+import { Hero } from './../hero';
+import { Component, OnInit } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { debounceTime, distinctUntilChanged, switchMap } from "rxjs/operators";
+
+@Component({
+  selector: 'app-hero-search',
+  templateUrl: './hero-search.component.html',
+  styleUrls: ['./hero-search.component.css']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/infinity-arc/Kobus-tour-of-heroes/pullRequest?prid=7838883)
```suggestion
  styleUrls: ["./hero-search.component.css"]
```